community/concepts/grain.md

```diff
@@ -0,0 +1,59 @@
+### Grain
+
+A big part of SourceCred’s vision is the desire to create project-specific cryptoassets, called "Grain" that represent contribution to or support of a project. These cryptoassets hold promise to make open-source projects financially sustainable which is currently an unsolved problem. You can read more about grain [here](concepts/grain.md).
+
+#### Properties of Grain
+
+ - Grain is always project-specific. For example, "SourceCred grain" and "Maker Community grain" would be wholly different assets.
+ - Every week, new grain is created, or "harvested" by the project.
```

Comment:
Wondering if we want to capitalize Grain? Uppercase implies a special noun, and distinguishes it from the more common use of the term. Thoughts @decentralion?